### ISSUE TITLE
Flagged as Malware by AV

### ISSUE THREAD

**Issue by slcraft** [sshnet/SSH.NET#887]
Installed SSH.NET 2020.0.1 as a nuget package in a clean Visual Studio C# project, and ran the following code - which immediately triggered multiple AntiVirus notifications which ended up quarantining the application.  

```
SftpClient sftp = new SftpClient("hostname", "user", new PrivateKeyFile(@"C:\Path\to\id_rsa"));
sftp.Connect();
```

The cited threat from AV was that it was acting like: not-a-virus:PDM:AdWare.NSIS.ConvertAd.bCb.

**Comment by IgorMilavec** [comment]
Should be fixed by #867

**Comment by IgorMilavec** [comment]
Duplicate of #797
